ember-welcome-page/rollup.config.mjs

```diff
@@ -22,11 +22,11 @@ export default {
     // not everything in publicEntrypoints necessarily needs to go here.
     addon.appReexports(['components/**/*.js']),
 
-    // This babel config should *not* apply presets or compile away ES modules.
-    // It exists only to provide development niceties for you, like automatic
-    // template colocation.
-    babel({
-      babelHelpers: 'bundled',
+    // compile TypeScript to latest JavaScript, including Babel transpilation
+    typescript({
+      transpiler: 'babel',
+      browserslist: false,
+      transpileOnly: false,
```

Comment:
Copy-pasted from the TypeScript project generated by `embroider-build/addon-blueprint`.